spikeextractors/extractors/neoextractors/blackrockextractor.py

```diff
@@ -31,8 +31,10 @@ class BlackrockRecordingExtractor(NeoBaseRecordingExtractor):
     installed = HAVE_NEO
     NeoRawIOClass = 'BlackrockRawIO'
 
-    def __init__(self, filename: PathType, block_index: Optional[float] = None, seg_index: Optional[float] = None):
-        super().__init__(filename=filename, block_index=block_index, seg_index=seg_index)
+    def __init__(self, filename: PathType, nsx_to_load: Optional[int] = None, block_index: Optional[float] = None, 
+                 seg_index: Optional[float] = None, **kwargs):
```

Comment:
`block_index` and `seg_index` should be int!
```suggestion
    def __init__(self, filename: PathType, nsx_to_load: Optional[int] = None, block_index: Optional[int] = None, 
                 seg_index: Optional[int] = None, **kwargs):
```